_includes/nav-bar.html

```diff
@@ -1,7 +1,9 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <div class="page-wide">
     <nav class="bottom-gap">
         <a class="brand-link" href="/home/" title="Go back Home"><img class="brand-logo" src="/images/graphics/bm-logo-transparent.png"></img></a>
         <div class="nav-pages">
+            <a href="/home/" class="nav home-text">Home</a>
```

Comment:
Home text to replace image when on small (mobile) screens. This is pretty basic but I just wanted to get in a quick solution to the navbar nastiness on mobile.